### PR TITLE
CIV-14145 Claimant LR screen Permission from the court

### PR DESCRIPTION
**Pull request by Omaira-Melo-Hmcts**
### JIRA link (if applicable) ###
[CIV-14145 Claimant LR screen: Permission from the court](https://tools.hmcts.net/jira/browse/CIV-14145)

### Change description ###
As a Claimant LR

I want to discontinue a claim in part or full

So that claim can be discontinued or case updated

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by German-A-Amoros — APPROVED** [review]

File: e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Feature('CCD Settle and discontinue claim 2v1 API test @api-spec');
+Scenario('Settle claim 2v1 scenario', async ({I, api_spec}) => {
+  await api_spec.createClaimWithRepresentedRespondent(config.applicantSolicitorUser, 'TWO_V_ONE');
+  await api_spec.createCaseFlags(config.hearingCenterAdminWithRegionId2);
+  await api_spec.manageCaseFlags(config.hearingCenterAdminWithRegionId2);
+  await api_spec.settleClaimSelectClaimant(config.applicantSolicitorUser, 'YES');
+});
+
+Scenario.only('Discontinue claim 2v1 scenario', async ({I, api_spec}) => {
```
Comment (ShwetaTandel-hmcts) [inline e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js:13]: Is this Scenario Only neeeded?

File: e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+
+const config = require('../../../config.js');
+
+Feature('CCD Settle and discontinue claim 2v1 API test @api-spec');
```
Comment (ShwetaTandel-hmcts) [inline e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js:5]: Should this be only non prod?

**Review by ShwetaTandel-hmcts — COMMENTED** [review]
Added comment on api tests

File: e2e/api/steps_LRspec.js [hunk 3]
```diff
@@ -1948,6 +1975,8 @@ const validateEventPages = async (data, solicitor) => {
     }
     if (pageId === 'OptionsForSettlement' || pageId === 'ClaimantDetails'){
       await assertValidDataSettleClaim(data, pageId);
+    } if (pageId === 'MultipleClaimant' || pageId === 'claimantConsent'){
```
Comment (hmcts-version1-ignacio) [inline e2e/api/steps_LRspec.js:1978]: Need to merge changes from CIV-14128

File: e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js (hunk 1 above)
Comment (Omaira-Melo-Hmcts) [reply e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js:13]: You are right deleted the Only from the test. My mistake, thanks @ShwetaTandel-hmcts

File: e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js (hunk 2 above)
Comment (Omaira-Melo-Hmcts) [reply e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js:5]: Added the tag

File: e2e/api/steps_LRspec.js (hunk 3 above)
Comment (Omaira-Melo-Hmcts) [reply e2e/api/steps_LRspec.js]: Merged

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by German-A-Amoros — DISMISSED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**